src/dialogs/examples/teams/UpdateCardMsgSetupDialog.ts

```diff
@@ -3,23 +3,35 @@ import { TriggerActionDialog } from "../../../utils/TriggerActionDialog";
 import { DialogIds } from "../../../utils/DialogIds";
 import { DialogMatches } from "../../../utils/DialogMatches";
 import { Strings } from "../../../locale/locale";
+import * as config from "config";
 
 export class UpdateCardMsgSetupDialog extends TriggerActionDialog {
-
+    // setup the card message and then user can update the card using below update dialog file
+    // microsoft-teams-sample-complete-node\src\dialogs\examples\teams\UpdateCardMsgDialog.ts
     private static async setupCardMessage(session: builder.Session, args?: any | builder.IDialogResult<any>, next?: (args?: builder.IDialogResult<any>) => void): Promise<void> {
-        let tempCard = new builder.HeroCard(session).title(Strings.loading);
-        let msg = new builder.Message(session).addAttachment(tempCard);
+        let updateCardCounter = 0;
+
+        let messageBackButtonValue = JSON.stringify({ updateCounterKey: updateCardCounter });
+        let messageBackButton = builder.CardAction.messageBack(session, messageBackButtonValue)
+            .displayText(Strings.messageBack_button_display_text)
+            .title(Strings.update_card_button)
+            .text("update card message");
```

Comment:
Add a comment here saying
// This must be a string that routes to UpdateCardMsgDialog, which handles card updates

Also in UpdateCardMsgDialog itself at line 20 (where you set the text of the messageBack action)

Reply:
Done.